backend/main/linting.py

```diff
@@ -1,28 +1,89 @@
 import ast
-
 from pyflakes import checker
-from pyflakes.messages import UnusedVariable
+from pyflakes.messages import UnusedImport, UnusedVariable, IsLiteral, RedefinedWhileUnused, ImportShadowedByLoopVar, \
+    ImportStarUsed, ImportStarUsage, ImportStarNotPermitted, DuplicateArgument, MultiValueRepeatedKeyLiteral
 
 MESSAGES = {
+    UnusedImport: """
+**Unused import `{0}`**
+You imported a module `{0}` but never used it. Did you forget to use it?
+Maybe you used the wrong import in its place? If you don't need it, just remove it entirely.
+    """,
     UnusedVariable: """
 **Unused variable `{0}`**
-
```

Comment:
Keep the blank lines so that the title shows on a different line from the body. A blank line separates paragraphs in markdown.

Reply:
Ok changed all of them to this format!